tests/features/common/actions/table.action.js

```diff
@@ -0,0 +1,71 @@
+import { expect } from 'chai'
+
+function getColumnValues(driver, table, columnName) {
+  return driver
+    .findElements(table.tableColumns[columnName])
+    .then(async function(elements) {
+      return await Promise.all(
+        elements.map(async function(element) {
+          return await element.getText()
+        })
+      )
+    })
+}
+
+const action = {
+  getColumnValues: getColumnValues,
+  isContainsValueInColumn: async function(driver, table, columnName, value) {
+    await getColumnValues(driver, table, columnName)
+      .then(arr => {
+        return arr.includes(value)
+      })
```

Comment:
If the entire function body is just one `return` statement, you can do this instead:
```suggestion
      .then(arr => arr.includes(value))
```